SUPP-14: set reasonably timeouts for fax_woker

icehess opened the pull request:
Timeouts in fax_worker stages was increased to the assumption
that it timing out and causing job to be retried when faxing
between accounts. As it turns out this was not the main issue
and the real fix is here https://github.com/2600hz/kazoo-core/pull/65 .

Now changing timeouts once more to give it a more relax time
if something taking more times to finish like sending the page.

lazedo commented on src/fax_worker.erl:
```diff
@@ -123,20 +123,21 @@ start_link(AccountId, JobId) ->
     Srv = props:get_value('server', Props),
-    gen_server:cast(Srv, {'tx_resp', kz_api:msg_id(JObj), JObj}).
+    JobId = props:get_value('job_id', Props),
+    gen_server:cast(Srv, {'tx_resp', JobId, JObj}).
```
we don't need this

lazedo commented on src/fax_worker.erl:
```diff
@@ -123,20 +123,21 @@ start_link(AccountId, JobId) ->
 handle_fax_event(JObj, Props) ->
     Srv = props:get_value('server', Props),
-    JobId = kz_call_event:authorizing_id(JObj),
+    JobId = props:get_value('job_id', Props),
```
same here

lazedo commented on src/fax_worker.erl:
```diff
@@ -123,20 +123,21 @@ start_link(AccountId, JobId) ->
     'true' = kapi_fax:query_status_v(JObj),
     Srv = props:get_value('server', Props),
-    JobId = kapi_fax:job_id(JObj),
+    JobId = props:get_value('job_id', Props),
```
same here

lazedo commented on src/fax_worker.erl:
```diff
@@ -382,7 +383,7 @@ handle_event(JObj, #state{job_id=JobId}) ->
         CallId -> kz_log:put_callid(<<JobId/binary, "|", CallId/binary>>)
     end,
-    {'reply', []}.
+    {'reply', [{'job_id', JobId}]}.
```
same here, we're already setting the login previous lines

icehess committed:
SUPP-14: set reasonably timeouts for fax_woker

Timeouts in fax_worker stages was increased to the assumption
that it timing out and causing job to be retried when faxing
between accounts. As it turns out this was not the main issue
and the real fix is here 2600hz/kazoo-core#65 .
